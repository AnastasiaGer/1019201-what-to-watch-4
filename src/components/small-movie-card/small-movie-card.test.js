import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card.jsx";

const movie = {
  title: `title-1`,
  image: `image-1`
};

describe(`SmallMovieCard`, () => {
  it(`Should render correctly movie card`, () => {
    const tree = renderer
      .create(<SmallMovieCard
        movie={movie}
        onTitleClick={() => {}}
        onCardHover={() => {}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
