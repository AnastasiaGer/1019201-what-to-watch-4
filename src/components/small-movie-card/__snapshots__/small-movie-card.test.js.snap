// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard Should render correctly movie card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    className="small-movie-card__link"
    href="/film/123890"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        className="player__video"
      />
      <img
        alt="Snatch"
        height="175"
        src="img/snatch.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      Snatch
    </h3>
  </a>
</article>
`;
